packages/import-notation/index.js

```diff
@@ -1,3 +1,23 @@
+const hashSet = require('hash-set');
+
+const tmpl = {
+    b : b => `b:${b}`,
+    e : e => e ? ` e:${e}` : '',
+    m : m => Object.keys(m).map(name => `${tmpl.mn(name)}${tmpl.mv(m[name])}`).join(''),
+    mn : m => ` m:${m}`,
+    mv : v => v.length ? `=${v.join('|')}` : '',
+    t : t => t ? ` t:${t}` : ''
+};
+
+const btmpl = Object.assign({}, tmpl, {
+    m : m => m ? `${tmpl.mn(m['name'])}${tmpl.mv([m['val']])}` : ''
+});
+
+const BemCellSet = hashSet(cell =>
+    ['block', 'elem', 'mod', 'tech']
```

Comment:
Why you can't use `cell.id`?

Reply:
I decided to fix this in different task: https://github.com/bem/bem-sdk/issues/265